samples/nats_jetstream/v3/receiver/main.go

```diff
@@ -0,0 +1,62 @@
+/*
+ Copyright 2024 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/nats-io/nats.go/jetstream"
+
+	cejsm "github.com/cloudevents/sdk-go/protocol/nats_jetstream/v3"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+func main() {
+	ctx := context.Background()
+
+	natsURL := "nats://localhost:4222"
+	natsSubject := "sample"
+
+	consumerOpt := cejsm.WithConsumerConfig(&jetstream.ConsumerConfig{FilterSubjects: []string{natsSubject}})
+	urlOpt := cejsm.WithURL(natsURL)
+	receiver, err := cejsm.NewReceiver(ctx, consumerOpt, urlOpt)
```

Comment:
Please rewrite your examples using the protocol methods and not sender/receiver implementations